component/service/src/main/java/org/exoplatform/social/service/malwareDetection/rest/MalwareDetectionRestService.java

```diff
@@ -28,21 +29,27 @@
 public class MalwareDetectionRestService implements ResourceContainer {
 
   private MalwareDetectionService malwareDetectionService;
+  private static final String MALWARE_DETECTION_PASSPHRASE_PROPERTY = "exo.malwareDetection.passphrase";
+  private static final String MALWARE_DETECTION_PASSPHRASE_PROPERTY_DEFAULT = "@malwareDetection/TOCHANGE!@";
   
   public MalwareDetectionRestService(MalwareDetectionService malwareDetectionService) {
     this.malwareDetectionService = malwareDetectionService;
   }
 
   @POST
-  @RolesAllowed("administrators")
   @Produces(MediaType.APPLICATION_JSON)
   @ApiOperation(value = "Process infected items",
       httpMethod = "POST",
       response = Response.class,
       notes = "This processes infected items")
   @ApiResponses(value = {
-      @ApiResponse(code = 200, message = "Request fulfilled") })
-  public Response processInfectedItems(List<String> infectedItemsPaths) throws Exception {
+      @ApiResponse(code = 200, message = "Request fulfilled"),
+      @ApiResponse(code = 403, message = "Unauthorized to process infected items")})
+  public Response processInfectedItems(List<String> infectedItemsPaths, @QueryParam("token") String token) throws Exception {
+    String malwareDetectionPassphrase = System.getProperty(MALWARE_DETECTION_PASSPHRASE_PROPERTY, MALWARE_DETECTION_PASSPHRASE_PROPERTY_DEFAULT);
```

Comment:
Should this property be read in constructor ? so that you dont have to reread it at each call